### DISCUSSION TITLE
Filter string from output

### DISCUSSION THREAD

**Discussion by jankete** [totaldebug/atomic-calendar-revive#656] [Feature Requests]
Hi, I use this card to display some birthdays via the contact calendar from google. There is always a "hat Geburtstag" in the event. It would be great, if I could hide this in the output.

![image](https://user-images.githubusercontent.com/22338766/167774132-35be24e2-b53d-4214-9ad6-9195c59a6d79.png)

I had this feature in this card: https://github.com/ljmerza/calendar-card/issues/80, but this project is discontinued.

**Comment by buteo72** [comment]
Hello, I like the feature as well, but then a bit different. It would be nice to substitute some words from the title with another string.
From MagicMorror: https://docs.magicmirror.builders/modules/calendar.html#configuration-options

titleReplace
An object of textual replacements applied to the tile of the event. This allow to remove or replace certain words in the title.
Example: {'Birthday of ' : '', 'foo':'bar'}

In my case the Calander title always has RN365 in front of the title. With this “titleReplace” it should be possible to remove / substitute certain strings in the title.
